sklearn/preprocessing/value_dropper.py

```diff
@@ -0,0 +1,216 @@
+# Author : Raghav RV <rvraghav93@gmail.com>
+#
+# Licence : BSD 3 clause
+
+import numpy as np
+
+from ..utils import check_X_y
+from ..utils import check_random_state
+
+__all__ = ["drop_values"]
+
+
+def drop_values(X, y=None, missing_mask=None,
+                missing_values="NaN",
+                missing_fraction=0.1,
```

Comment:
This is not documented below. Also, could be called `drop_fraction` or `fraction`.